src/Statistics.jl

```diff
@@ -0,0 +1,257 @@
+using Random: shuffle!
+using StatsBase: AbstractWeights, Weights, values, varcorrection
+@doc doc"""
+    mean(M,x; weights=Weights(ones(n),n), x0=x[1], stop_iter=100, kwargs... )
+
+computes the Riemannian center of mass also known as Karcher mean of the vector
+`x` of `n` points on the [`Manifold`](@ref) `M`. This function
+uses the gradient descent scheme. Optionally one can provide
+weights $w_i$ for the weighted Riemannian center of mass.
+The general formula to compute the minimizer reads
+````math
+\argmin_{y\in\mathcal M} \frac{1}{2}\sum_{i=1}^n w_i\mathrm{d}_{\mathcal M}^2(y,x_i),
```

Comment:
I think you need this
```suggestion
\operatorname{argmin}_{y\in\mathcal M} \frac{1}{2}\sum_{i=1}^n w_i\mathrm{d}_{\mathcal M}^2(y,x_i),
```
Also perhaps should have a `\sum_i=1^n w_i` term in here to avoid confusion re the weights.

Reply:
On my side the \argmin renders perfectly fine? Even with half a space between the g and the m as it should be

Reply:
Okay no worries then. Haven't render on my machine, was just using LaTeXiT.

Reply:
It'S defined within `mathtools` or something that is loaded/implemented in MathJax.